lib/extensions.js

```diff
@@ -81,16 +81,43 @@ function getBinaryName() {
  * is set, return that path. Otherwise make
  * path using current release version and
  * binary name.
+ * 
+ * Optionally, the default URL can be replaced by
+ * setting the environment variable SASS_BINARY_SITE
+ * (or commandline option --sass-binary-site)
+ * which should point to the mirror of the repository
+ * laid out as follows:
+ *
+ * SASS_BINARY_SITE/
+ * 
+ *  v3.0.0-beta.4
+ *  v3.0.0-beta.4/freebsd-x64-14_binding.node
+ *  v3.0.0-beta.5
+ *  v3.0.0-beta.5/freebsd-ia32-11_binding.node
+ *  v3.0.0-beta.5/freebsd-x64-42_binding.node
+ *  v3.0.0-beta.5/freebsd-x64-14_binding.node
+ *  v3.0.0-beta.5/freebsd-x64-11_binding.node
+ *  v3.0.0-beta.5/freebsd-ia32-43_binding.node
+ *  v3.0.0-beta.5/freebsd-ia32-42_binding.node
+ *  v3.0.0-beta.5/freebsd-ia32-14_binding.node
+ *  v3.0.0-beta.5/freebsd-x64-43_binding.node
+ *
+ * ... etc. for all supported versions and platforms
  *
  * @api private
  */
 
 function getBinaryUrl() {
   return flags['--sass-binary-url'] ||
-         package.nodeSassConfig ? package.nodeSassConfig.binaryUrl : null ||
+         (package.nodeSassConfig ? package.nodeSassConfig.binaryUrl : null) ||
          process.env.SASS_BINARY_URL ||
```

Comment:
Now that we have sassBinarySite, we do not need sassBianryUrl and the test in api.js that goes with it.